swagger.yaml

```
@@ -117,6 +117,12 @@ paths:
           description: "Runs query through NLP processors to influence search weighting"
           type: boolean
           required: false
+        - name: uri_prefix
+          in: query
+          description: URI prefix to filter the search results
```

I think it would be useful to show that the URI prefix should begin with a forward slash e.g. 
```suggestion
          description: URI prefix to filter the search results e.g. /economy
```